intel_pytorch_extension_py/__init__.py

```diff
@@ -3,6 +3,4 @@
 from .version import __version__
 from .optim import *
 from .ops import *
-import _torch_ipex as core
-
-core._initialize_aten_bindings()
+import _torch_ipex as core
```

Comment:
better integrate these two packages into one, so that end  user only needs to import one package.